test/lib/ansible_test/_data/completion/docker.txt

```diff
@@ -1,7 +1,7 @@
 default name=quay.io/ansible/default-test-container:1.10.1 python=3.6,2.6,2.7,3.5,3.7,3.8 seccomp=unconfined
 centos6 name=quay.io/ansible/centos6-test-container:1.8.0 python=2.6 seccomp=unconfined
 centos7 name=quay.io/ansible/centos7-test-container:1.8.0 python=2.7 seccomp=unconfined
-centos8 name=quay.io/ansible/centos8-test-container:1.10.0 python=3.6 seccomp=unconfined
+centos8 name=quay.io/ansible/centos8-test-container:latest python=3.6 seccomp=unconfined
```

Comment:
Is it a good idea to get rid of an explicit version number here? Or is this just temporary?

Reply:
Just temporary for testing. I just update that image, which @pabelanger found fixed the issue in his testing.